cylc/flow/task_pool.py

```diff
@@ -719,46 +732,87 @@ def rh_release_and_queue(self, itask) -> None:
             self.queue_task(itask)
 
     def _get_spawned_or_merged_task(
-        self, point: 'PointBase', name: str, flow_nums: 'FlowNums'
-    ) -> Optional[TaskProxy]:
+        self, point: 'PointBase', tdef: 'TaskDef', flow_nums: 'FlowNums'
+    ) -> 'Tuple[Optional[TaskProxy], bool, bool]':
         """Return new or existing task point/name with merged flow_nums"""
```

Comment:
Would be good to have a quick description of the two new return values for future reference:

```suggestion
        """Return new or existing task point/name with merged flow_nums
        
        Returns:
            tuple - (itask, is_in_pool, is_xtrig_sequential)

            itask:
                The requested task proxy, or None if task does not
                exist or is not valid at the specified point.
            is_in_pool:
                TODO
            is_xtrig_sequential:
                TODO

        """
```